engine/access/rpc/backend/script_executor.go

```diff
@@ -81,7 +94,7 @@ func (s *ScriptExecutor) Initialize(indexReporter state_synchronization.IndexRep
 //   - storage.ErrHeightNotIndexed if the data for the block height is not available. this could be because
 //     the height is not within the index block range, or the index is not ready.
 func (s *ScriptExecutor) ExecuteAtBlockHeight(ctx context.Context, script []byte, arguments [][]byte, height uint64) ([]byte, error) {
-	if err := s.checkDataAvailable(height); err != nil {
+	if err := s.checkHeight(height); err != nil {
```

Comment:
I assume that checkHeight now returns a different error, not `storage.errHeightNotIndexed` meaning we need to update godoc for all functions that depend on it.